<launch>
  <param name="/use_sim_time" value="true" />

  <arg name="input_topic" default="/cartographer_ros/merge_point_cloud" />
  <arg name="baselink_support" default="false" />
  <node pkg="mmdet_pp" type="mmdet_pp"
    name="mmdet_pp" output="screen" >
    <remap from="/points_raw" to="$(arg input_topic)"/>
    <param name="baselink_support"            value="$(arg baselink_support)" />
  </node>

  <rosparam command="load" file="$(find object_tracking)/config/object3d_detector.yaml"/>
  <node pkg="object_tracking" type="object_tracking" name="object_tracking_01" output="screen"/>

  <!-- <node pkg="detected_objects_visualizer" type="visualize_detected_objects" name="point_pillars_visualization_01"
        output="screen" ns="/detection/lidar_detector" /> -->

</launch>